packages/components/src/confirm-dialog/component.tsx

```diff
@@ -23,10 +23,10 @@ import { VStack } from '../v-stack';
 import * as styles from './styles';
 import { useCx } from '../utils/hooks/use-cx';
 
-function ConfirmDialog(
-	props: WordPressComponentProps< OwnProps, 'div', false >,
+const ConfirmDialog = (
```

Comment:
For consistency, let's call this unexported component `UnconnectedConfirmDialog` and the exported component `ConfirmDialog`

```diff
diff --git a/packages/components/src/confirm-dialog/component.tsx b/packages/components/src/confirm-dialog/component.tsx
index 7f3b62fe33..dbe5bf5f90 100644
--- a/packages/components/src/confirm-dialog/component.tsx
+++ b/packages/components/src/confirm-dialog/component.tsx
@@ -23,9 +23,9 @@ import { VStack } from '../v-stack';
 import * as styles from './styles';
 import { useCx } from '../utils/hooks/use-cx';
 
-const ConfirmDialog = (
-	props: WordPressComponentProps< OwnProps, 'div' >,
-	forwardedRef: ForwardedRef< any >
+const UnconnectedConfirmDialog = (
+	props: WordPressComponentProps< OwnProps, 'div', false >,
+	forwardedRef: React.ForwardedRef< any >
 ) => {
 	const {
 		isOpen: isOpenProp,
@@ -122,5 +122,9 @@ const ConfirmDialog = (
 	);
 };
 
-const ConnectedConfirmDialog = contextConnect( ConfirmDialog, 'ConfirmDialog' );
-export default ConnectedConfirmDialog;
+const ConfirmDialog = contextConnect(
+	UnconnectedConfirmDialog,
+	'ConfirmDialog'
+);
+
+export default ConfirmDialog;

```

Reply:
Done.